src/test/objectstore/test_bluefs.cc

```diff
@@ -620,6 +651,45 @@ TEST(BlueFS, test_replay) {
   rm_temp_bdev(fn);
 }
 
+TEST(BlueFS, test_replay_growth) {
+  uint64_t size = 1048576LL * (2 * 1024 + 128);
+  TempBdev bdev{size};
```

Comment:
This is causing a make check failure https://jenkins.ceph.com/job/ceph-pull-requests/63024/consoleFull#-15327964252a811ea2-3e7b-466b-84b4-d13df7e35809